### PR TITLE
Add Request test 

### PR DESCRIPTION
**Pull request by jheart-vic**
## What I did
- Created a Request spec file for all my controllers.
- Made sure that for each action I checked:
    - If the response status was correct.
    - If a correct template was rendered.
    - If the response body includes correct placeholder text.

### CODE REVIEW

File: spec/requests/posts_spec.rb [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    Comment.create(user: @user, post: @post, text: 'Hello bob')
+    Comment.create(user: @user, post: @post, text: 'Hello gram')
+    Comment.create(user: @user, post: @post, text: 'Hello sam')
+  end
+  describe 'GET /index' do
+    before(:example) { get user_posts_path(@user) } # get(:index)
+    it 'assigns all posts to @posts' do
+      get user_posts_path(@user)
+      expect(assigns(:posts)).to eq(Post.all)
+    end
```
Comment (codepantha) [inline spec/requests/posts_spec.rb]: - [ ] This particular test fails as shown in the image below. Please try to fix it 👇 
image

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: REQUIRES CHANGES ♻️ 

## Hi @jheart-vic 👋

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights 🎯 
✔️ Descriptive PR
✔️ You have tested your controllers
✔️ Good commit history
✔️ No linter errors

## Required Changes ♻️
_Check the comments under the review._

- [ ] I noticed that the MIT license link in your README is broken. Kindly fix it as it is important to distribute your software with a license🙏 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/requests/posts_spec.rb (hunk 1 above)
Comment (jheart-vic) [reply spec/requests/posts_spec.rb]: No it doesn't fail for me

**Review by Munsa1 — APPROVED** [review]
Hi @jheart-vic, 

congratulations-2



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow

## Optional suggestions
- [ ] Kindly remove the two pending test suites.
image

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**